app/models/conference.rb

```diff
@@ -329,8 +329,10 @@ def event_distribution
   # * +hash+ -> hash
   def scheduled_event_distribution
     confirmed_events = program.events.where(state: 'confirmed')
-    scheduled_value =  { 'value' => confirmed_events.where.not(start_time: nil).count, 'color' => 'green' }
-    unscheduled_value =  { 'value' => confirmed_events.where(start_time: nil).count, 'color' => 'red' }
+    confirmed_scheduled_events = confirmed_events.reject{|a| a.event_schedules.empty?}
```

Comment:
Sorry my bad, actually we already have a method to check if the event is scheduled.